umh-core/pkg/config/manager.go

```diff
@@ -112,6 +112,34 @@ func (m *FileConfigManager) GetConfig(ctx context.Context, tick uint64) (FullCon
 	return config, nil
 }
 
+// WriteConfig writes the given config to disk
```

Comment:
please document here, that it can lead to race conditions (if that is the correct word) here.

if you have a ctx with a deadline, and the filesystem operation takes too long, it will cancel with an error, but might still execute the underlying WriteFile system oepration.